orttraining/orttraining/python/training/onnxblock/optim/optim.py

```diff
@@ -161,9 +161,10 @@ def build(self, parameters):
         # TODO: Avoid hard coded input/output strings
         learning_rate_name = "learning_rate"
         step_name = "step"
+        params_name = "params"
```

Comment:
is that possible we fix those pylint issues? that is a bit annoying for reviewing that code :)

Reply:
or for some non-sense issues, we can suppress them with some annotation.

Reply:
I tried to make some changes in order to reduce the plint warnings. Let me know if it is better now.